jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -271,14 +273,22 @@ private void extractTar() {
   }
 
   private void assignLocalImageResult(Future<LocalImage> localImage) {
-    results.baseImageLayers = executorService.submit(() -> localImage.get().layers);
     results.baseImageAndRegistryClient =
```

Comment:
I didn't bother in the last PR as the overall implementation is in the transitioning state from a single image to multiple images, but given the current state, I think we should rename this to baseImage**s**AndRegistryClient to be consistent.